src/Symfony/Component/Serializer/Tests/Normalizer/ObjectNormalizerTest.php

```diff
@@ -740,6 +764,11 @@ class ObjectOuter
     private $inner;
     private $date;
 
+    /**
+     * @var ObjectInner[]
+     */
+    private $inners;
```

Comment:
If there is no type defined (through a PHPDoc or any other way supported by registered extractors), the old behavior (don't check anything) occurs.